examples/demo-rollup/src/config.rs

```diff
@@ -51,6 +61,10 @@ mod tests {
                     path: PathBuf::from("/tmp"),
                 },
             },
+            rpc_config: RpcConfig {
+                bind_host: "127.0.0.1".to_string(),
```

Comment:
Do we need them to specify the host? Or can we default to "127.0.0.1"?

Reply:
oh, the rollup_config.toml does have it set to 127.0.0.1 so it should default to that
the "127.0.0.1" here is for the test case.
i think sometimes users like binding to 0.0.0.0 or even a specific network interface

Reply:
Hmm, yeah good point. We might want to add a serde default to 127.0.0.1 so that users can just not specify an interface if they want that, but that's out of scope - and you're right that it's nice to have the option to configure something else